tests/test_study.py

```diff
@@ -428,7 +437,7 @@ def f(trial):
         # non-nested: 5, params: 2, user_attrs: 1 system_attrs: 1
         assert len(df.columns) == 9
         for i in range(3):
-            assert ('trial_id', '') in df.columns  # trial_id depends on other tests.
+            assert ('number', '') in df.columns  # trial_id depends on other tests.
```

Comment:
The word "trial_id" in the comment should be replaced with "trial_number".

Reply:
Thank you for pointing it out.
Actually, 'trial_number' does not depends on other studies, and I will rewrite this assertion as follows:

```python
df.number[i] == i
```